### PR TITLE
[TEST] Add User Authentication Tests in BACKEND

### LINKED ISSUE #28: [TEST] Add User Authentication Tests in BACKEND
Add tests for logging in and signing up.

### PR DESCRIPTION
**Pull request by YungiKim1999**
- setup babel and jest to test the backend of the application
- added test cases for the User Authentication endpoints
- closes #28

### CODE REVIEW

File: backend/src/routes/__tests__/auth.test.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import mongoose from 'mongoose';
+import express from 'express';
+// import axios from 'axios';
```
Comment (afei088) [inline backend/src/routes/__tests__/auth.test.js:4]: remove this if its not being used

File: frontend/.eslintrc.json [hunk 2]
```diff
@@ -2,7 +2,8 @@
     "env": {
         "browser": true,
         "es2021": true,
-        "jest": true
+        "jest": true,
```
Comment (afei088) [inline frontend/.eslintrc.json:5]: why is this needed in the frontend?

Comment (YungiKim1999) [reply frontend/.eslintrc.json:5]: for some reason, it fails building if I don't have it
I do not know if this is something I introduced with testing but when I removed it, the lining would not work

Comment (afei088) [reply frontend/.eslintrc.json:5]: you can keep jest, but jasmine can be removed

**Review by afei088 — APPROVED** [review]
